Apps/Admin/Client/Pages/Support.razor.cs

```diff
@@ -126,7 +126,7 @@ public MessagingVerificationRow(MessagingVerificationModel model)
 
                 };
                 this.Verified = model.Validated ? "true" : "false";
-                this.VerificationDate = DateFormatter.ToShortDateAndTime(model.UpdatedDateTime);
+                this.VerificationDate = DateFormatter.ToShortDateAndTime(model.UpdatedDateTime.ToLocalTime());
```

Comment:
We can't use ToLocalTime as the servers are on UTC.  You have to have specific conversion logic or have the front-end to the conversion (recommended).

Reply:
Yes.

Reply:
I moved the toLocalTime and date formatter code to the front end Support.razor code instead.

Reply:
Did you try doing a custom deploy to the mock environment for confirmation?

Reply:
No I haven't.
I have a question though.  Is it true that the .razor.cs code runs on the server machine and the .razor code runs on the client?  If that is true, as Stephen is suggesting, then the DashboardPage.razor.cs code: "private int TimeOffset { get; set; } = (int)TimeZoneInfo.Local.GetUtcOffset(DateTime.Now).TotalMinutes;" would not give back the correct offset if the server is on UTC time.

Reply:
As discussed, this is client code so your original code was fine.  Sorry